src/Components/TodoList.jsx

```diff
@@ -42,9 +42,14 @@ const TodoList = ({ tasks, getTasksFromFirebase }) => {
                       <div>
                         <label>{todo.name}</label>
                         <p className="text-xs">{dateConvert(todo.date)}</p>
+                        {todo.dueDate.length == 0 ? null : (
```

Comment:
Here you can use the Logical AND operator: 
`{todo.dueDate.length && (<p>{formatDueDate(todo.dueDate)}</p>
                        )}`

If todo has a dueDate than return the paragraph

**! Important:** Never user **==** (equality), instead use **===** (strict equal) to avoid misbehaving